source/life_expectancy_script.R

```diff
@@ -0,0 +1,142 @@
+library(tidyverse)
+library(readxl)
+
+file_path <- "data/life-expectancy/life-expectancy-17-19-tab3.xlsx"

```

Comment:
I'd maybe move the spreadsheet up one level so it sits next to the main spreadsheet